util.js

```diff
@@ -52,3 +52,36 @@ exports.getTestModifiers = function getTestModifiers(node) {
 
 	return [];
 };
+
+const getMembers = node => {
+	const name = node.property.name;
+
+	if (node.object.type === 'MemberExpression') {
+		return getMembers(node.object).concat(name);
+	}
+
+	return [name];
+};
+
+exports.getMembers = getMembers;
+
+const assertionMethodsNbArguments = {
```

Comment:
I would name it: `assertionMethodNumArgs`. Num is a more common short name for Number.